WordPress/UITests/Tests/LoginTests.swift

```diff
@@ -84,7 +79,7 @@ class LoginTests: XCTestCase {
 
             // Login flow returns MySites modal, which needs to be closed.
             .closeModal()
-
-        XCTAssert(MySiteScreen.isLoaded())
+            .verifyMySiteScreenLoaded()
+            .removeSelfHostedSite()
```

Comment:
`logOutDefaultWordPressComAccount` works for WordPress accounts; needed to add this additional step for self-hosted sites for this work.